docs/tutorial/5-relationships-and-hyperlinked-apis.md

```diff
@@ -106,6 +106,8 @@ If we're going to have a hyperlinked API, we need to make sure we name our URL p
 
 After adding all those names into our URLconf, our final `snippets/urls.py` file should look something like this:
 
+    from django.conf.urls import url, include
```

Comment:
I tend to agree... If that's the case, a few other samples show a diff, versus all imports. While I was going through the samples, I would copy/paste and review in my text editor. So it was difficult when diffs were provided. Anyone else have opinions?